dashboard/app.py

```diff
@@ -0,0 +1,98 @@
+"""
+Streamlit app that runs the Dashboard.
+"""
+import logging
+import webbrowser
+
+import bcrypt
+from dotenv import load_dotenv
+import extra_streamlit_components as stx
+from extra_streamlit_components.CookieManager import CookieManager
+import pandas as pd
+import streamlit as st
+
+from cookies import set_cookies, clear_cookies_of_session
+from database import get_database_connection, load_all_database_info, get_user_info
+from rendering import render_dashboard, render_login_page
+
+
```

Comment:
This is great! Thank you for wiring this. 

Not sure why it isn't working on your laptop - we'll investigate.